liquibase-dist/src/main/archive/GETTING_STARTED.txt

```diff
@@ -117,29 +117,18 @@ Running the update command allowed your developer database to match the defined
 
 In this example, we will add a new changeSet to create a "works for" column in the persons table.
 
-To add this changeSet, open the sample changelog file in your existing editor, and uncomment the following changeSet:
+To add this changeSet, open the sample changelog file in your existing editor, and copy + paste one of the following changeSet examples:
 
   XML changelog example code
   ---------------------------
-    <changeSet id="3" author="your.name">
-        <addColumn tableName="person">
-            <column name="worksfor_company_id" type="int"/>
-        </addColumn>
-    </changeSet>
     <changeSet id="4" author="your.name">
-        <addForeignKeyConstraint constraintName="fk_person_worksfor"
-                                 baseTableName="person" baseColumnNames="worksfor_company_id"
-                                 referencedTableName="company" referencedColumnNames="id"/>
+        <addForeignKeyConstraint constraintName="fk_person_worksfor" baseTableName="person" baseColumnNames="worksfor_company_id" referencedTableName="company" referencedColumnNames="id"></addForeignKeyConstraint>
     </changeSet>
 
   SQL changelog example code
   ---------------------------
-
-    --changeset your.name:3
-    ALTER TABLE person ADD worksfor_company_id INT;
-
-    --changeset your.name:4
-    ALTER TABLE person ADD CONSTRAINT fk_person_worksfor FOREIGN KEY (worksfor_company_id) REFERENCES company(id);
+      --changeset your.name:4
+    ALTER TABLE person ADD FOREIGN KEY (fk_person_worksfor) REFERENCES company(id) ;
```

Comment:
I think we need this fully left-justified because if people simply copy/paste it without removing the leading spaces it may cause problems.

Reply:
Justified :)